lib/zendesk_api/collection.rb

```diff
@@ -541,9 +542,13 @@ def array_method(name, *args, &block)
       to_a.public_send(name, *args, &block)
     end
 
+    # If you call client.tickets.foo - and foo is not an attribute nor an association, it ends up here, as a new collection
     def next_collection(name, *args, &block)
       opts = args.last.is_a?(Hash) ? args.last : {}
-      opts.merge!(:collection_path => @collection_path.dup.push(name))
+      opts.merge!(collection_path: @collection_path.dup.push(name), page: nil)
```

Comment:
Minor:

```suggestion
      opts.merge!(collection_path: [*@collection_path, name], page: nil)
```

It's more declarative (no `dup.push`).